### ISSUE TITLE
Feature Request: Support for signed/negative decimal values

### ISSUE THREAD

**Issue by apolkosnik** [gchq/CyberChef#176]
### Summary
I had a small array of signed decimals e.g. -4,-24,-119,0... and I wanted to convert them into hexadecimals.

"To Decimal" throws an error.

Perhaps it would be great to have a checkbox for "signed" or some way to convert to unsigned e.g.:

```
var x = [ -4,-24,-119,0];
x.forEach(function(s) { 
     document.write((s < 0 ? (0xFF + s + 1) : s).toString(16));
} );
```

**Comment by bwhitn** [comment]
[(https://gchq.github.io/CyberChef/#recipe=Fork(',',',',false)Conditional_Jump('%5E%5C%5Cs*(%5C%5Cd%2B)',4,10)Regular_expression('User%20defined','%5E%5C%5Cs*-(%5C%5Cd%2B)',true,true,false,'List%20capture%20groups')From_Decimal('Space')XOR(%7B'option':'Hex','string':'FF'%7D,'Standard',false)To_Charcode('Space',10)Merge()From_Decimal('Comma')To_Hex('Comma')&input=LTQ0LCAtNiwgNTUsIC0yMiwgMzM)]
This is long and complex but it works. Maybe some better number parsing and math operations would fix this?

**Comment by n1474335** [comment]
Closed in #390.
